src/pages/projectSettings/tabs/utils/types.ts

```diff
@@ -10,5 +10,6 @@ export enum ProjectType {
 export const PatchTriggerAliasStatus = {
```

Comment:
are the two "success" options intentional / temporary?
<img width="423" alt="Screenshot 2023-08-20 at 12 11 46 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/a4e10b44-b8aa-4d9d-a8ad-d1e86ce10b3f">